webapp/components/Avatar/Avatar.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-  <ds-avatar :image="avatarUrl" :name="userName" class="avatar" :size="size" />
+  <ds-avatar
+    :image="user && user.avatar | proxyApiUrl"
```

Comment:
I needed to check what the `|` means in this case @roschaefer 
it's a bitwise OR, correct?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators#.7c_%28Bitwise_OR%29
can you explain why you used it in this case?

Reply:
No, this is not a bitwise OR. It's the use of a vue filter in a template. See https://vuejs.org/v2/guide/filters.html